eng/pipelines/latest-jdk.yml

```diff
@@ -60,9 +60,12 @@ stages:
 
           - template: /eng/pipelines/templates/steps/install-reporting-tools.yml
 
+          - pwsh: |
+              Write-Host "##vso[task.setvariable variable=IsLatestNonLtsJdk;]true"
```

Comment:
This can be moved up into `variables`

Reply:
I tried doing that and it complained about it. I can test that out again.